src/System.Security.Principal.Windows/src/PinvokeAnalyzerExceptionList.analyzerdata.netcoreapp

```diff
@@ -0,0 +1 @@
+advapi32.dll!LsaNtStatusToWinError
```

Comment:
That's interesting, it means this isn't in onecore but apparently the API's that need it are. OK to submit but let me check into that.